apisix/plugins/openid-connect.lua

```diff
@@ -49,6 +49,7 @@ local schema = {
 local _M = {
     version = 0.1,
     priority = 2599,
+    type = 'auth',
```

Comment:
ditto

you can create a new PR to fix this

Reply:
I'm sorry but I currently can't afford the time to split this into separate PRs. I think the change set is reasonably small to be considered in one PR. If you prefer more fine-grained PRs, then please consider creating these yourself, e.g. based on the changes in this branch.

Reply:
I found you have submitted new PR: https://github.com/apache/apisix/pull/2905/files

Many thanks for your nice job.

Small PR is important, it easy to find problems and we can merge it asap when it is small.

By the way, it is a clearer way to show what we are doing.

Many thx for nice contribution again @jenskeiner

Reply:
Sure, you're very welcome. Thanks for the info. I'll try to put the remaining outstanding changes from this one into a separate new PR and then we take it from there.

Reply:
Have created #2910 which should contain the outstanding changes from this PR.